mailet/test/src/main/java/org/apache/mailet/base/test/FakeMailContext.java

```diff
@@ -133,6 +133,11 @@ public Builder sender(MailAddress sender) {
                 return this;
             }
 
+            public Builder sender(Optional<MailAddress> sender) {
+                sender.ifPresent(this::sender);
```

Comment:
could be in one line
```java
return sender.map(this::sender).orElse(this);
```

Reply:
I do not like your line as it IMO hide the fact that we are always returning `this`

(and also that we are mutating the state of the current builder:

 - ifPresent is **expected** to have side effects, not **map**

Hence I decline this proposal.